src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -113,6 +107,13 @@ private function handle(?array $payload)
                     }
                 }
             }
+        } elseif ('Create' === $payload['type']) {
+            $actor = $this->manager->findActorOrCreate($payload['actor']);
+            if ($actor instanceof Magazine && $actor->lastOriginUpdate < (new \DateTime())->modify('-3 hours')) {
```

Comment:
```suggestion
            if ($actor instanceof Magazine && (!$actor->lastOriginUpdate || $actor->lastOriginUpdate < (new \DateTime())->modify('-3 hours'))) {
```

Reply:
modifying datetime rather than the field actually seems to protect us from exceptions, I think you had commented on it before when I had asked, as `null < now - 3 hours` will be `true`. I was contemplating changing all the `!var || var->modify comparison datetime` to `var comparison datetime->modify`

Reply:
True, then you can ignore that comment